docs: review bt mesh models (#2504)
nrfconnect/sdk-nrf: doc/nrf/templates/mesh_model_main_README.rst

.. tip::
   The Common types section is optional.

>    If the model does not have its own common types, mention in this section that the model does not have its own common types.

Reviewer: ```suggestion
   Use this section to inform if the model does not have its own common types.
```
